prow/cmd/hook/main.go

```diff
@@ -152,9 +151,11 @@ func main() {
 		logrus.WithError(err).Fatal("Error getting git client.")
 	}
 
+	logger := logrus.StandardLogger()
```

Comment:
I've left hook out for now because I didn't feel strong in adding a stanza for it in the config but eventually we may need to as other things come up.